_includes/doc/admin-guide/options/channel-args.md

```diff
@@ -0,0 +1,15 @@
+## channel-args()
+
+|  Type:|      |
```

Comment:
could you plz format these tables with spaces to see the table structure (equal-width columns, fields, here, and everywhere bellow as well)

Reply:
fixed